CUDLR/Scripts/Console.cs

```diff
@@ -14,7 +15,25 @@ struct QueuedCommand {
   }
 
   public class Console {
-
+     
+     /* Struct containing console message and color data */
+	 struct ConsoleMessage
+      {
+          public string message;
+          public Color32 color;
```

Comment:
Looking at the docs for Color32 would it have been better to use Color class instead? Would this cause any problems?  I ask because there are some static defaults for Color on the class which you could use, and you can still get the RGBA with color[0] ... color[3].